libbeat/tests/system/beat/beat.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, args, outputfile, env={}):
     def start(self):
         # ensure that the environment is inherited to the subprocess.
         variables = os.environ.copy()
-        variables = variables.update(self.env)
+        variables.update(self.env)
```

Comment:
this was wrong, `dict.update` returns nothing

Reply:
You are correct, I am really to used to ruby that always returned the modified values by default.

I've created https://github.com/elastic/beats/pull/8998 with your change, It need to go in 6.4 6.5, 6.x